deploy/overlays/custom/kustomization.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Kustomization for custom mixer.
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+nameSuffix: -custom
+namespace: mixer
+
+bases:
+- ../../base
+
+resources:
+- serviceaccount.yaml
+
+configMapGenerator:
+- name: mixer-configmap
+  behavior: create
+  literals:
+    - projectId=<PROJECT_ID>
+    - serviceName=<DOMAIN>
+- name: store-configmap
+  behavior: create
+  files:
+  - bigtable.version
```

Comment:
Should we remove bigtable.version, since external people won't have it?

Reply:
The base deployment.yaml requires "bigtable.version" config. we can separate bt and bq to two config but I don't want to make it too complicated since this is a noop for custom deploy